src/pages/MonthlyGood/MoreAboutKiva.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div class="more-about-kiva-section-wrapper">
+		<div class="row">
+			<div class="small-12 columns">
+				<h2 class="impact-text">
+					More about Kiva
+				</h2>
+			</div>
+			<div class="small-12 large-7 columns">
+				<p>
```

Comment:
It might be possible to put the column classes on this paragraph, and the following paragraph, and then get rid of the wrapping div

Reply:
I can go either way on this. I think the template looks a little cleaner with the grid classes on the divs.

Reply:
It does look a little cleaner, but mobile performance is greatly dependent on the number of dom elements in the page, so we should take every chance we can to use as few elements as possible